Application/Setting/Univention/UniventionUser.php

```diff
@@ -0,0 +1,280 @@
+<?php
+namespace SPHERE\Application\Setting\Univention;
+
+use SPHERE\Application\Setting\Univention\Service\Entity\TblUnivention;
+use SPHERE\System\Extension\Repository\Debugger;
+
+class UniventionUser
+{
+    private $curlhandle;
+    private $server;
+    private $token;
+
+    public function __construct() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Opening brace should be on a new line](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)